src/web/app/components/question-responses/per-question-view-responses/per-question-view-responses.component.ts

```diff
@@ -67,4 +74,35 @@ export class PerQuestionViewResponsesComponent implements OnInit, OnChanges {
     this.responsesToShow = responsesToShow;
   }
 
+  sortResponses(by: SortBy): void {
```

Comment:
I think there is a sort.serviece integrated recently.

Reply:
I followed the implementation similar to `InstructorHomePageComponent` and `StudentListInfoTableComponent`. I don't think these have been refactored into sort services?